packages/fxa-settings/src/styles/unit-row.scss

```diff
@@ -7,6 +7,8 @@
 
   &-header {
     @apply font-header w-full mb-1;
+    min-width: 165px;
```

Comment:
there's some screen widths where this doesn't work
<img width="306" alt="Screen Shot 2020-11-04 at 9 41 47 AM" src="https://user-images.githubusercontent.com/87619/98151278-c421d500-1e84-11eb-8de9-365e9ac64b57.png">

here's a diff that works better
```diff
diff --git a/packages/fxa-settings/src/styles/unit-row.scss b/packages/fxa-settings/src/styles/unit-row.scss
index 980a82d29..f03b824ff 100644
--- a/packages/fxa-settings/src/styles/unit-row.scss
+++ b/packages/fxa-settings/src/styles/unit-row.scss
@@ -27,21 +27,22 @@
   &-hr {
     @apply mx-5 border-grey-100;
   }
 }
 
 @screen mobileLandscape {
   .unit-row {
     @apply flex-no-wrap px-6;
 
     &-header {
-      @apply flex-1 mb-0 mr-2;
+      @apply flex-none mb-0 mr-2;
+      width: 10rem;
     }
 
     &-content {
       @apply flex-2;
     }
 
     &-actions {
       @apply flex-1 flex justify-end;
     }
   }
```